universal-login-relayer/lib/core/services/messages/ExecutionWorker.ts

```diff
@@ -28,18 +29,28 @@ class ExecutionWorker {
   }
 
   async loop() {
-    do {
-      const nextMessage = await this.executionQueue.getNext();
-      if (nextMessage && this.messageExecutor.canExecute(nextMessage)){
+    const tick = async () => {
+      if (this.state === 'stopping'){
+        this.state = 'stopped';
+      } else {
+        await sleep(this.tick);
+      }
+    };
```

Comment:
Should we move/extract function to class?

Reply:
Addressed by https://github.com/UniversalLogin/UniversalLoginSDK/pull/1278